Resources/Prototypes/_Floof/Loadouts/Jobs/Medical/uncategorized.yml

```
@@ -347,3 +347,19 @@
     - !type:CharacterDepartmentRequirement
       departments:
         - Medical
+
+- type: loadout
+  id: LoadoutMedicalUniformJumpskirtNurse
+  category: JobsMedicalAUncategorized
+  cost: 0
+  exclusive: true
+  items:
+  - ClothingUniformJumpskirtNurse
+  requirements:
+  - !type:CharacterItemGroupRequirement
+    group: LoadoutMedicalUniforms
+  - !type:CharacterJobRequirement
+    jobs:
+    - MedicalDoctor
+    - MedicalIntern
+    - SeniorPhysician
```

Unless this is intended to be restricted to just these 3 jobs, I recommend:
```suggestion
  - !type:CharacterDepartmentRequirement
    departments:
    - Medical
```

---

It is because I wasn't sure a chemist, paramed, or CMO would want to be a nurse, but I can certainly change it if you think it's wise